tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -41,6 +41,10 @@ var (
 	createdInstanceCreds *types.Credentials
 	// Temp directory ansible created on the windows host
 	ansibleTempDir = ""
+	// kubernetes-node-windows-amd64.tar.gz SHA512
+	// Value from https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.16.md#node-binaries-1
+	expectedSha = "a88e7a1c6f72ea6073dbb4ddfe2e7c8bd37c9a56d94a33823f531e303a9915e7a844ac5880097724e44dfa7f4a9659" +
```

Comment:
why is this being broken up into different strings? So it doesnt hit the 120 char limit in our style guidelines?

Reply:
Correct